Library/Homebrew/extend/pathname.rb

```diff
@@ -131,6 +131,12 @@ def write(content, *open_args)
     open("w", *open_args) { |f| f.write(content) }
   end
 
+  # Only appends to a file that is already created.
+  def write_append(content, *open_args)
+    raise "Cannot append file that doesn't exist: #{self}" unless exist?
+    open("a", *open_args) { |f| f.puts(content) }
```

Comment:
I'm not enamoured with the alternative name, but I'll take it if it's the only thing holding this up at this point.